src/main/java/cn/nukkit/network/protocol/ResourcePackStackPacket.java

```diff
@@ -9,7 +9,7 @@
     public static final byte NETWORK_ID = ProtocolInfo.RESOURCE_PACK_STACK_PACKET;
 
     public boolean mustAccept = false;
-    public ResourcePack[] behaviourPackStack = new ResourcePack[0];
+    public ResourcePack[] behaviorPackStack = new ResourcePack[0];
```

Comment:
I'm not sure if you are aware but it isn't misspelled, that's the way it is spelled in British English.